types/experiments.ts

```diff
@@ -0,0 +1,72 @@
+/** All fields are publically accessible via the API and available to clients */
+export interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  public_name: string;
+  /** Short public description of the experiment */
+  public_description: string;
+  /** Experimenter URL */
+  experiment_url: string;
+
+  /** Is the experiment currently live in production? i.e., published to remote settings? */
+  is_published: boolean;
+  /** Are we continuing to enroll new users into the experiment? */
+  is_enrollment_paused: boolean;
+
+  /** Reference to an Audience, which contains targeting information */
+  audience: Audience;
+  /** Bucketing configuration */
+  bucket_config: BucketConfig;
+
+  /** A list of cusom feature metrics relevant to the experiment analysis */
+  feature_metrics: Array<FeatureMetric>;
+  /** Branch configuration for the experiment */
+  branches: Array<Branch>;
+
+  /** Actual publish date of the experiment */
+  start_date: Date;
+  /** Actual end date of enrollment */
+  enrollment_end_date: Date;
+  /** Actual end date of the experiment */
+  end_date: Date;
+}
+
+interface BucketConfig {
+  randomization_unit: "client_id" | "normandy_id";
+  namespace: string;
+  start: number;
+  count: number;
+  /**
+   * Total number of buckets
+   * @default  */
+  total: number;
+}
+
+export interface Audience {
+  /** Human-readable name of the audience */
+  name: string;
+  /** Human-readable description */
+  description: string;
+  /** A targeting expression written in jexl */
+  filter_expression: string;
+}
+
+interface Branch {
+  /** Display name for variant, e.g. "control"  */
+  slug: string;
+  /** Is this a reference/"control" branch? */
+  is_reference_branch: string;
+  /**
+   * Relative ratio of population for the branch (e.g. if branch A=1 and branch B=3,
+   * branch A would get 25% of the population)
+   * @default 1
+   */
+  ratio: number;
+  /** The variant payload. TODO: This will be more strictly validated. */
+  value: {[key: string]: any};
+}
```

Comment:
So the design schema in https://github.com/mozilla/rapid-experiments-shared/pull/25 will have to conform to this schema?

Reply:
Yes and no – there might be some instructions for the UI that fall outside of this schema, but for definign default values, yeah, it should conform

Reply:
Is there a separate schema for the design data?

Reply:
Updated in #25 to have the preset use an experiment type schema, which extends the `ExperimentsInternal`